torch/onnx/__init__.py

```diff
@@ -124,7 +124,8 @@ def symbolic_override_first_arg_based(symbolic_fn):
 
     def might_trace(args):
         import torch
-        return torch._C._jit_is_tracing(args[0])
+        first_arg = args[0]
+        return torch.is_tensor(first_arg) and torch._C._jit_is_tracing(first_arg)
```

Comment:
I don't think we should silently skip if it's not a tensor, because it's likely an error